tide_publication.module

```diff
@@ -25,53 +23,6 @@ function tide_publication_entity_bundle_create($entity_type_id, $bundle) {
   /** @var \Drupal\Core\Extension\ModuleHandlerInterface $moduleHandler */
   $moduleHandler = \Drupal::service('module_handler');
 
-  if ($entity_type_id == 'node' && in_array($bundle, [
```

Comment:
Removed all these, because it also runs in the install hook, so don't see the point of running it twice.